libraries/mysql_service_manager_sysvinit.rb

```diff
@@ -39,7 +39,7 @@ class MysqlServiceManagerSysvinit < MysqlServiceBase
       end
 
       service mysql_name do
-        provider Chef::Provider::Service::Init::Redhat if node['platform_family'] == 'redhat'
+        provider Chef::Provider::Service::Init::Redhat if %w(redhat suse).include?(node['platform_family'])
```

Comment:
I believe we did this for sure when Chef 11 did the wrong thing, but will Chef do the right thing when we're trying to force it to use a non-native init system?